services/api/src/config.ts

```diff
@@ -64,6 +65,7 @@ nconf.argv()
   PORT: 8081,
   SEQUELIZE_LOGGING: true,
   SUPER_USER_IDS: [],
+  SEQUELIZE_SSL: true,
```

Comment:
Setup default to true, so that we don't have to do any changes in the production